CMSgov/eAPD
# updated Delete FFY modal

resolves #2996 

**Description-**
Changed the FFY delete alert to match the rest of the delete modals.

**This pull request changes...**

- changed the text for the Delete FFY modal to match the mock up
- changed ApdSummary to use the modal instead of the window confirm
- updated ApdSummary tests to use RTL instead of enzyme
- added userEvent module to make tests easier to write


**Steps to manually verify this change...**

1. Login
2. Open APD

Test adding
- Click 2023
- Expected: The 2023 FFY should be checked

Test removing and canceling 
- Click 2020
- Expected: The delete modal to pop up and match the modal
- Click Cancel
- Expected: The delete modal should close and 2020 should still be checked

Test removing
- Click 2021
- Expected: The delete modal to pop up and match the modal
- Click Delte
- Expected: The delete modal should close and 2021 should not be checked

### This pull request is ready to review when...


### This pull request can be merged when…
